{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/users/auth/serializers.py

```diff
@@ -21,7 +23,7 @@ class RegisterSerializer(serializers.Serializer):
     password = serializers.CharField(required=True)
 
     class Meta:
-        fields = ['email', 'name']
+        fields = ['email', ]
```

Comment:
Reason to modify this? i.e why remove `name` field from the existing serializer?

Reply:
name does not exist as a field in the register serializer. It subclasses serializers.Serializer.
Also, since both the fields are `required`, we can do away with the field declaration in the Meta class. I have tested it. Thoughts ?

Reply:
remove the `fields` meta.